lib/appium_lib/device/device.rb

```diff
@@ -18,7 +18,10 @@ module Device
         device_time:            'session/:session_id/appium/device/system_time',
         current_activity:       'session/:session_id/appium/device/current_activity',
         current_context:        'session/:session_id/context',
-        get_network_connection: 'session/:session_id/network_connection'
+        get_network_connection: 'session/:session_id/network_connection',
```

Comment:
I just add new commit [here](https://github.com/appium/ruby_lib/pull/400/commits/9ef723c07b4b5d2f95775f20979d3f9c438845cf).
Selenium client has commands but it has no bridge to use them. So, I just add bridges in https://github.com/appium/ruby_lib/pull/400/commits/9ef723c07b4b5d2f95775f20979d3f9c438845cf . (and I remove endpoint which I added before.)